plugins/azad-proxy.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: azad-proxy
+spec:
+  platforms:
+  - uri: https://github.com/XenitAB/azad-kube-proxy/releases/download/v0.0.3/kubectl-azad_proxy-darwin-amd64.zip
+    sha256: 51ae0098cbbb0b5da01f5b086c02474a8b74012e5f373c2dbeea36ad659c65e2
+    bin: kubectl-azad_proxy
+    files:
+    - from: kubectl-azad_proxy
+      to: .
+    - from: LICENSE
+      to: .
+    selector:
+      matchLabels:
+        os: darwin
+        arch: amd64
+  - uri: https://github.com/XenitAB/azad-kube-proxy/releases/download/v0.0.3/kubectl-azad_proxy-linux-amd64.zip
+    sha256: 323182ba7960e3b587c5d7b6971e0aaad29c0d5d1acb4d276e20c7f7c5c3ab98
+    bin: kubectl-azad_proxy
+    files:
+    - from: kubectl-azad_proxy
+      to: .
+    - from: LICENSE
+      to: .
+    selector:
+      matchLabels:
+        os: linux
+        arch: amd64
+  - uri: https://github.com/XenitAB/azad-kube-proxy/releases/download/v0.0.3/kubectl-azad_proxy-windows-amd64.zip
+    sha256: 5556aa4a6b679fe8e33639fd9df008290f2a3b9f0b965dc2fcdde0c913721233
+    bin: kubectl-azad_proxy.exe
+    files:
+    - from: kubectl-azad_proxy.exe
+      to: .
+    - from: LICENSE
+      to: .
+    selector:
+      matchLabels:
+        os: windows
+        arch: amd64
+  version: "v0.0.3"
+  shortDescription: Plugin to generate and handle authentication for azad-kube-proxy
```

Comment:
"Plugin to" redundant. It's clear this is a plugin :)

Reply:
@ahmetb fixed!